### DISCUSSION TITLE
I cannot imagine, why GitHub think, that my image is manipulated!

### DISCUSSION THREAD

**Discussion by HugyakAndi** [community/community#143727] [GitHub Education]
### Hi everyone,

Hi,
I am student at a Hungarian University and GitHub refused my application twice with the message, that my student card image is malipulated. But of course it is not!
Please bes o kind and let me know, how can I solve this problem? I need the student package to complete my diplom project asap, I am very sad, because of this procedure.
Thank oyu very much!

**Comment by BbySharp-dev** [comment]
Hi there,

I'm sorry to hear you're having trouble with the GitHub Student Developer Pack application process. It sounds frustrating, especially since you need it for your diploma project.

Here are a few steps you can try to resolve the issue:

Ensure Document Clarity: Make sure that your student card photo is clear, well-lit, and that all details are easily readable. Sometimes even slight issues in lighting or resolution can cause an application to be flagged.

Avoid Editing the Image: If you’ve used any software to edit or crop the photo, try uploading the unedited original. Some automatic detection systems can misinterpret edited photos as “manipulated.”

Contact GitHub Support Directly: If you’re certain your documentation is correct, consider reaching out directly to GitHub Education support. You can open a support request here: [GitHub Education Support](https://support.github.com/contact/education). Explain your situation in detail, including that you’re a student in Hungary and need the pack for your diploma project.

Provide Additional Documents: If possible, have a secondary proof of enro

Hopefully, these steps will help you get approved. Good luck with your diploma project, and don’t hesitate to re

Best of luck!

**Reply by HugyakAndi** [reply]
Hi, okay, I will try it, thank you very much for the prompt answer, I really appreciate it.
I used Gimp to decrease the image size, because the original was larger, than 1GB, which is also denied by GitHub.
